lib/StatusConnector.php

```diff
@@ -68,11 +68,13 @@ abstract public function getStatus();
      */
     public static function getBadgeByStatus($status)
     {
-        if ($status === OK) {
+        $statusAsInt = intval($status);
+
+        if ($statusAsInt === 0) {
```

Comment:
Maybe we could keep the constants as they were, just change the value they represent? I think reading through the code and seeing "WARNING" instead of "1" is much more clear.